models/app.model.js

```diff
@@ -56,3 +56,10 @@ exports.selectArticlesByQuery = () => {
     });
 };
 
+exports.removeCommentByID= (id) =>{
+    return db.query(`DELETE FROM comments
+    WHERE comment_id = $1 RETURNING *;`, [id]).then((result)=>{
+        return result.rows.length ? result.rows[0] : Promise.reject({status : 404, msg: 'Not Found'})
```

Comment:
You have chosen to return something here and I don't think we need to do that as we would be doing nothing with that comment that has been deleted. We have a 204 status code and by default, express will not send anything back on the response body. 

If would be enough to check if there is a rows.length and then just return a rejected promise if it's length is 0